packages/admin/admin/src/form/FinalFormFileUpload.tsx

```diff
@@ -122,6 +153,23 @@ const FinalFormFileUploadComponent: React.FunctionComponent<WithStyles<typeof st
             </div>
         ));
 
+    const rejectedFiles = fileRejections.length > 0 && (
+        <>
+            {fileRejections.map((rejectedFile) => (
+                <div key={rejectedFile.file.name} className={classes.rejectedFileListItem}>
+                    {rejectedFile.file.name.length < 20 ? rejectedFile.file.name : `${rejectedFile.file.name.substring(0, 20)}...`}
```

Comment:
Is it possible to implement the `...` effect with `text-overflow: ellipsis;`?

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow?retiredLocale=de

Reply:
There is no maxWidth for the element, so there is no overflow, the element just stretches. Therefore I implemented it with "...", so that all file items have the same length.

Reply:
@jamesricky what do you think? Is there a way to achieve this with CSS only?

Reply:
<img width="344" alt="Screenshot 2024-02-22 at 12 52 19" src="https://github.com/vivid-planet/comet/assets/6264317/c1d87242-8956-4657-bec7-31e4e25fdde4">

Is that this text here?
Shouldn't be a problem with `text-overflow: ellipsis`

Reply:
I suppose it is. @NataliaVizintini what do you think?

Reply:
`text-overflow` wasn't working because `max-width` in `FieldContainer` wasn't set. @jamesricky helped me to make it work.